src/pyflex/window_selector.py

```diff
@@ -420,6 +452,8 @@ def reject_based_on_signal_to_noise_ratio(self):
         noise = self.observed.data[self.config.noise_start_index:
                                    self.config.noise_end_index]
 
+        offset = self.event.origin_time - self.observed.stats.starttime
```

Comment:
> One more thing, for the method "reject_based_on_signal_to_noise_ratio", I feel like we should provide people with one more option that uses both criteria, "amplitude" and "energy". At least for my case, I would like to have both checked... People can also chose which mode they want to use based on the features of their data.

:+1: Could just be called `both` or `amplitude_and_energy` - I prefer the later one as its more descriptive.